docs/docs/reconciliation.md

```diff
@@ -52,7 +47,6 @@ It is very unlikely that a `<Header>` element is going to generate a DOM that is
 
 As a corollary, if there is a `<Header>` element at the same position in two consecutive renders, you would expect to see a very similar structure and it is worth exploring it.
 
-
 ### DOM Nodes
 
 When comparing two DOM nodes, we look at the attributes of both and can decide which of them changed in linear time.
```

Comment:
Nit: can we use className instead of id attribute? You'd almost never use id attribute in React, and one can potentially confuse it with key.